File: java/1_tipeDataDanVariable/BilanganBulat.java

class BilanganBulat{
    
    public static void main(String[] args){
        byte b = 10;
        short s = 11;
        int i = 12;
        long l = 13;

        System.out.printf("Byte b bernilai:     %d dengan range: %d Byte",b, Byte.BYTES); System.out.println();
        System.out.printf("Short s bernilai:    %d dengan range: %d Byte",s, Short.BYTES); System.out.println();
        System.out.printf("Int i bernilai:      %d dengan range: %d Byte",i, Integer.BYTES); System.out.println();
        System.out.printf("long l bernilai:     %d dengan range: %d Byte",l, Long.BYTES); System.out.println();
    }
}

/*
 * byte ->  1Byte
 * short -> 2Byte
 * int ->   4Byte
 * long ->  8Byte
 */

File: kotlin/1_tipeDataDanVariable/BilanganBulat.kt
 
 fun main(){
    val b: Byte = 10
    val s: Short = 11
    val i: Int = 12
    val l: Long = 13
    
    println("Byte   b bernilai:   $b dengan range:  ${Byte.SIZE_BYTES}  Byte")
    println("Short  s bernilai:   $s dengan range:  ${Short.SIZE_BYTES} Byte")
    println("Int    i bernilai:   $i dengan range:  ${Int.SIZE_BYTES}   Byte")
    println("long   l bernilai:   $l dengan range:  ${Long.SIZE_BYTES}  Byte")
}

/*
 * Byte ->  1Byte
 * Short -> 2Byte
 * Int ->   4Byte
 * Long ->  8Byte
 */